packages/react/src/components/Link/index.css

```
@@ -3,3 +3,6 @@ a.Button--secondary {
   text-decoration: none;
   padding: 9px 18px;
 }
+a.Button--thin {
+  padding: 0px 16px;
```

Nit: Values of 0 in css can usually be unitless

```suggestion
  padding: 0 16px;
```